cli/endpoints/online/triton/multi-models/multi-endpoint-aks.yml

```diff
@@ -0,0 +1,21 @@
+name: aks-multi-model
+type: online
+target: azureml:aks-gpu-deploy
+auth_mode: Key
+traffic:
+    etblue: 100
+deployments:
+    - name: etblue
+      model: azureml:multi-models:2
+      code_configuration:
+          code:
+             local_path: ./code
+          scoring_script: dummy.py
+      environment:
+            name: aks-multi-env
+            version: 1
+            docker:
+                image: metricstestacr.azurecr.io/triton-base:3
```

Comment:
In all yaml files, update environment.docker.image to mcr.microsoft.com/azureml/tritonserver-21.02-py38-inference:20210525.1

Reply:
Done as per comments